input-github-username.sh

```diff
@@ -0,0 +1,16 @@
+set -u;
+if ! gp env | grep -q 'GITHUB_USERNAME'; then {
+  until read -r -p "++++ Enter your Github username (only required once): " GITHUB_USERNAME; do continue; done
+  gp env GITHUB_USERNAME="$GITHUB_USERNAME"; # Sets this variable on https://gitpod.io/variables which can be used later on
+} fi
+
+if test ! -v GITHUB_USERNAME; then {
+  eval $(gp env -e);
+} fi
+
+
+git -C /workspace/mattermost-server remote add upstream https://github.com/mattermost/mattermost-server.git
```

Comment:
You can use variables instead of hardcoding things.
I think I also had given an example for this in our discord thread 😉 
```
git -C "$GITPOD_REPO_ROOT" https://github.com/mattermost/mattermost-server.git

Reply:
Yeah I saw your example in discord thread. However, this wouldn't work for us because I am setting remote for other repos from root repo.